### ISSUE TITLE
Filament selection on Prepare tab

### ISSUE THREAD

**Issue by JeffreyBrew** [bambulab/BambuStudio#2192]
**Is your feature request related to a problem? Please describe.**
The way filaments are displayed on the Prepare page of Bambu Studio can get confusing.  Sequentially number filaments is not a good idea.  Filaments should be numbered in the same manner as on the Device page.  In the picture titled "Filaments as shown on Prepare" there are nine filaments displayed.  Filaments 8 and 9 are actually loaded in D3 and D4, respectively.

**Describe the solution you'd like**
I'd like to see the representations of the AMS as shown in "Filaments as shown on Device" also displayed on the Prepare page of Bambu Studio.  When a user clicks on one of the AMS, the filament(s) loaded into that AMS are shown below numbered accordingly (A1, A2, A3, A4, B1, B2, B3, B4, C1, C2, C3, C4, D1, D2, D3, D4) as shown in "Proposed Filament selection for Prepare (A)" and "Proposed Filament selection for Prepare (B)".  All other parts of Bambu Studio (e.g., Flushing volumes, etc.) should also be updated to use the same filament numbering.

**Describe alternatives you've considered**

**Additional context**
Filaments as shown on Prepare
![Filaments as shown on Prepare](https://github.com/bambulab/BambuStudio/assets/27747977/981b35b9-7c75-4804-bea3-2993c133fe0f)

Filaments as shown on Device
![Filaments as shown on Device](https://github.com/bambulab/BambuStudio/assets/27747977/d574b6e5-6a5b-4995-b417-a3deba78f74e)

Proposed Filament selection for Prepare (A)
![Proposed Filament selection for Prepare (A)](https://github.com/bambulab/BambuStudio/assets/27747977/59c46217-a990-42ad-9809-847628cfdbdd)

Proposed Filament selection for Prepare (B)
![Proposed Filament selection for Prepare (B)](https://github.com/bambulab/BambuStudio/assets/27747977/bab0b5f6-6fed-4240-a70a-3ac45add6586)

**Comment by Panda631** [comment]
Good idea

**Comment by Sonicboom247** [comment]
I agree, I was just searching for the same issue for the AMS lite for the A1. The AMS Lite is defined as 
1 and 4
2 and 3
![Screenshot 2024-01-15 093259](https://github.com/bambulab/BambuStudio/assets/84334417/7c800f7d-0fb2-414a-a4be-184af199189c)

whereas in Bambu Studio they are displayed as 
1 and 2
3 and 4
![Screenshot 2024-01-15 093010](https://github.com/bambulab/BambuStudio/assets/84334417/6e1629e8-48c0-489c-be68-74a83a01ea64)
These need to be represented like for like so i am not constantly verifying slot placement.
